chromium_src/components/variations/variations_seed_store_unittest.cc

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) 2025 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#include "src/components/variations/variations_seed_store_unittest.cc"
+
+namespace variations {
+
+// Verifies that VariationsSeedStore::UpdateSeedDateAndMaybeCountry() sets
+// prefs::kVariationsSeedDate preference on the first request.
+TEST_F(VariationsSeedStoreTest, UpdateSeedDateOnFirstRequest) {
```

Comment:
Verified that tests in current file are running with chromium unit tests: `npm run test components_unittests Component`